setup.py

```diff
@@ -28,7 +28,7 @@ def read(*filenames, **kwargs):
     author_email='zheller@gmail.com',
     version=about['__version__'],
     install_requires=[
-        'flake8',
+        'flake8>=3.5',
```

Comment:
Can you explain this change a little more?

Reply:
For some reason resolved `flake8` version was 3.2
It didn't work with `3.8`

The first version of `flake8` which worked is `3.5`

Reply:
Weird, I'd expect the `requirements-dev.txt` to supercede this. It looks like in fact, that is what was happening and this change wasn't respected in Travis CI:

(I think you mean `flake8@3.3`, not `3.2` for Python 3.8)

https://github.com/zheller/flake8-quotes/pull/90/commits/750f4a0f6bda934014970d2f48638c2417eb3f52

https://travis-ci.org/github/zheller/flake8-quotes/jobs/660092929#L191

The following change where you updated `requirements-dev.txt` installs appropriately

https://travis-ci.org/github/zheller/flake8-quotes/jobs/660095678#L191

https://github.com/zheller/flake8-quotes/pull/90/commits/795cbac666bf3c34dd54b0c63459ae543f8f763d

Can you rollback this line change?

Reply:
Looks like I can edit the file in the PR directly so going to do that for speedier turnaround